.env

```diff
@@ -3,10 +3,10 @@ VUE_APP_FRONTEND=localhost:8080
 
 VUE_APP_BACKEND_PLIOS=plios/
 VUE_APP_BACKEND_ITEMS=items/
+VUE_APP_BACKEND_VIDEOS=videos/
```

Comment:
I think we can remove these api endpoints from here and instead keep a config file (maybe config.js) as these endpoints are unlikely to change.

`.env` file should only contain variables that are supposed to differ on various environments.

Reply:
Agreed, I remember you suggested this before as well
Will do it